arangod/Cluster/DropDatabase.cpp

```diff
@@ -65,8 +67,9 @@ bool DropDatabase::first() {
   try {
     DatabaseGuard guard("_system");
     auto vocbase = &guard.database();
+    OperationOptions opOptions(ExecContext::current());
 
-    _result = Databases::drop(vocbase, database);
+    _result = Databases::drop(vocbase, database, opOptions);
```

Comment:
question why do you also use the operationoption here, this isn't a CRUD operation ?

Reply:
Fair. I got so used to passing around `OperationResult` that I didn't stop to think about the semantics of `OperationOptions`. I guess it makes more sense in some of these cases to just pass the `ExecContext` and a `Result` where necessary.